tests/nestedfor.c

```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
```

Comment:
A nested loop is a loop within a loop, an inner loop within the body of an outer one, and `for.c` is expected to verify a nested loop. Please go ahead.